### ISSUE TITLE
fels for S2 L2?

### ISSUE THREAD

**Issue by LauraZof** [vascobnunes/fetchLandsatSentinelFromGoogleCloud#55]
Hi! thanks for the script! It works well and is very useful
I wonder if there's a way to adapt fels to download L2 data instead of L1

thanks in advance
Laura

**Comment by vascobnunes** [comment]
Thanks for using it! 
I'm not sure if such data already exists on google cloud

**Comment by LauraZof** [comment]
yes, the data is available at https://console.cloud.google.com/storage/browser/gcp-public-data-sentinel-2;tab=objects?_ga=2.92285845.928848343.1620736884-1907755448.1619611486&prefix=&forceOnObjectsSortingFiltering=false

in the folder L2
cheers,
Laura

**Comment by vascobnunes** [comment]
Thank you! This in fact would be very interesting, however, I currently lack on availability to develop such feature. I'll keep it this issue open and in the wishlist.

**Comment by cjakes** [comment]
If you want to download Level 2 data, change the metadata souce at line 169 of fels.py

SENTINEL2_METADATA_URL = 'http://storage.googleapis.com/gcp-public-data-sentinel-2/L2/index.csv.gz'

Downloads fine however some of the fels options may break.
